src/plugins/orangehrmDashboardPlugin/test/fixtures/testcases/EmployeeOnLeaveAPITestCases.yaml

```diff
@@ -198,3 +212,4 @@ GetAll:
         startTime: null
     meta:
       total: 3
+      leavePeriod: null
```

Comment:
better to test leave period not null scenario also.
Check how Devishke deal with multiple fixture files;
https://github.com/orangehrm/orangehrm/pull/1432/files#diff-e1485c5df69716e96570bd9472d5efb6d4869b4b0a7ac1fe448712cfca653de8R31